src/ShinyDeploy/Domain/Deployment.php

```diff
@@ -188,6 +199,21 @@ public function deploy(bool $listMode = false): bool
         $this->logResponder->log('Running tasks...');
         if ($listMode === false && $this->runTasks('after') === false) {
             $this->logResponder->error('Running tasks failed. Aborting job.');
+
+            $isRepoStatusCorrupted = $this->checkRepoStatusIsCorrupted();
```

Comment:
Why is the repository check (and possible reset) executed a second time at this point?